index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/eb94992e2c.js" crossorigin="anonymous"></script> 
+
+</head>
+<body>
+    <header>
+        <div id="popup" class="pop">
+            <button id="cancel-icon" type="button">&times;</button>
+            <ul class="menu">
+              <li class="item" id="restoreMobileHome">Home</li>  
+              <li class="item" id="removeMobileHome">About</li>
+              <li class="item">Program</li>     
+              <li class="item">Join</li>
+              <li class="item">Sponsor</li>
+              <li class="item">News</li>
+            </ul>
+        </div>
+        <span id="mobileMenu" class="fas fa-bars fa-3x"></span>
+        <div id="desktop-menu" class="blackbg" >
+            <span><img src="media/icons8-facebook.svg" alt="facebook" width="60" height="100"></span>
+            <span><img src="media/icons8-twitter.svg" alt="twitter" width="100" height="100"></span>
+            <p  class="gray">English</p>
+            <p class="gray">My Page</p>
+            <p class="gray">Logout</p>
+        </div>
+        <nav id="desktop-nav" class="whitebg">
+           <div class="nav-logo">
+            <img src="media/headphone.png" alt="logo" id="restoreHome">
+           </div>
+           <div class="nav-contents">
+            <p class="red cursor" id="removeHome">About</p>
+            <p class="black cursor">Program</p>
+            <p class="black cursor">Join</p>
+            <p class="black cursor">Sponsor</p>
+            <p class="black cursor">News</p>
+            <button class="campaign red whitebg cursor" type="button">Campaign</button>
+            </div>
+        </nav>
+        <div id="header-homepage-section" class="heading">
+            <p class="company">Record Label & Music streaming</p>
+            <h1 id="main-heading">STRAY BEATZ</h1>
+            <p class="introduction black">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Dolorem adipisci, 
+               neque quisquam fugit officiis voluptates magnam amet ea ex molestiae quibusdam omnis
+            </p>
+    
+            <h2 class="time black">2022.10.15(THU) ~ 16(FRI)</h2>
+            <h3 class="venue black">@ Lekki phase 2, Lagos Nigeria</h3>
+        </div>    
+        <section id="header-about-section"> 
```

Comment:
- [ ] The footer in about section should have the logo container and should have a white background for mobile view: 

Your design  | Behance design
------------- | -------------
<img width="453" alt="Screenshot 2022-03-11 at 18 32 25" src="https://user-images.githubusercontent.com/77038610/157887377-f977ed8a-d4e2-447f-b892-4125b9fbbc4c.png">  | <img width="454" alt="Screenshot 2022-03-11 at 18 32 46" src="https://user-images.githubusercontent.com/77038610/157887424-015d60f7-e83c-4044-b3b8-49db12d07d23.png">

Reply:
The page has to be refreshed when viewed in developer mode(mobile view) in order to render correctly.
![Screenshot from 2022-03-11 19-33-14](https://user-images.githubusercontent.com/92347494/157898290-8e84577c-0e8a-45d7-a3c7-01dcf3b56ab1.png)

Reply:
I can see that it works when refreshed🙌